README.md

```diff
@@ -39,19 +40,22 @@ Describe the rules of scope in JavaScript.
 
 Your Answer:
 ```text
+There's global scope and local scope. Everything outside of a function is in the global scope, and only functions can create local scope. Variables defined in the global scope can be accessed inside a function, but variables defined inside a function cannot be accessed in the global scope, or in another function outside function where variable was defined.
 ```
```

Comment:
:+1: 
This looks good.  I want to be sure you are aware of "chaining of scopes". Any function defined within another function has a local scope which is linked to the outer function.  http://www.coolcoder.in/2014/03/everything-you-need-to-know-about.html